src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/CHIPNativeBridge.java

```diff
@@ -24,9 +24,9 @@
 
   // TODO: Add instance ref for CHIPDeviceController here
 
-  static {
-    System.loadLibrary("native-lib");
-  }
+  // static {
```

Comment:
Could we delete code instead of commenting it out?

Reply:
+1

Reply:
Reverted for now. This file is going to have follow up changes that will address this removal.